package.json

```diff
@@ -195,11 +195,12 @@
 		"glob": "7.1.2",
 		"husky": "7.0.0",
 		"inquirer": "7.1.0",
-		"jest": "27.4.5",
+		"jest": "29.3.1",
```

Comment:
Should we also bump some peer deps, like here:

https://github.com/WordPress/gutenberg/commit/caab0febf95760b6e4de61cc7663d0d7fa5c149d

I've noticed that being very flexible with dependencies can cause issues with more complex upgrades (upgrading `@wordpress` packages for example). Maybe it won't be the case with `jest` though.